Mathlib/Analysis/Complex/Circle.lean

```diff
@@ -159,3 +177,47 @@ protected lemma norm_smul {E : Type*} [SeminormedAddCommGroup E] [NormedSpace 
   rw [Submonoid.smul_def, norm_smul, norm_eq_of_mem_sphere, one_mul]
 
 end Circle
+
+namespace Real
+
+/-- The additive character from `ℝ` onto the circle, given by `fun x ↦ exp (2 * π * x * I)`.
+Denoted as `𝐞` within the `Real.FourierTransform` namespace. This uses the analyst convention that
+there is a `2 * π` in the exponent. -/
+def fourierChar : AddChar ℝ Circle where
+  toFun z := .exp (2 * π * z)
+  map_zero_eq_one' := by rw [mul_zero, Circle.exp_zero]
+  map_add_eq_mul' x y := by rw [mul_add, Circle.exp_add]
+
+@[inherit_doc] scoped[FourierTransform] notation "𝐞" => Real.fourierChar
+
+open FourierTransform
+
+theorem fourierChar_apply (x : ℝ) : 𝐞 x = Complex.exp (↑(2 * π * x) * Complex.I) :=
+  rfl
+
+@[continuity]
+theorem continuous_fourierChar : Continuous 𝐞 := Circle.exp.continuous.comp (continuous_mul_left _)
+
```

Comment:
In the same way there is `probChar_ne_one`, could you add `fourierChar_ne_one` here, to keep the APIs parallel?